library/src/ui/connections.js

```diff
@@ -0,0 +1,81 @@
+window.CoNDeT.ui.ConnectionsComponent = (function () {
+  var constructor = function () {};
+
+  constructor.prototype = Object.create(window.CoNDeT.ui.BaseComponent);
+
+  constructor.prototype.onInit = function (common, props) {
+    var element = document.createElement("svg");
```

Comment:
Powinno być [createElementNS](https://developer.mozilla.org/en-US/docs/Web/API/Document/createElementNS). I to samo tyczy się wszystkich elementów z svg (to powodowało ten dziwny błąd, że svg miał zły rozmiar, że strzałka się nie pojawiała itd)